src/sentry/middleware/integrations/parsers/msteams.py

```diff
@@ -46,19 +46,36 @@ def get_integration_from_request(self) -> Integration | None:
 
     def get_response(self) -> HttpResponseBase:
         if self.view_class not in self.region_view_classes:
+            logger.info(
+                "sentry.middleware.integrations.parsers.msteams: View class not in region",
+                extra=self.request_data,
```

Comment:
You might want to add one extra level of indirection here.

There are keys in `extra` that are reserved, and may overwrite values in `self.request_data`.

```suggestion
                extra={"request_data": self.request_data},
```

Reply:
done!